src/org/webonise/perimetercalculator/PerimeterCalculator.java

```diff
@@ -0,0 +1,18 @@
+package org.webonise.perimetercalculator;
+
+public class PerimeterCalculator implements PolygonCalculator{
+    private int noOfSides;
+    private Double sideLength,perimeter;
```

Comment:
Avoid declaring variables like this it should be  private Double sideLength,
                                                                           private Double perimeter;